### ISSUE TITLE
Accept configuration splits

### ISSUE THREAD

**Issue by NatoBoram** [android-police/apkmirror-public#107]
It's possible to export `.apks` from a device using [https://github.com/Aefyr/SAI/](SAI). It would be amazing if it was possible to upload those to APKMirror (https://github.com/Aefyr/SAI/issues/59), either as one bundle or as multiple splits. It would be even more interesting if APKMirror had a mobile client that can upload those or select the appropriate one when downloading.

**Comment by archon810** [comment]
We're actively working on support for split apks. More soon!

**Comment by NatoBoram** [comment]
Now that you invented your own proprietary `.apkm` format to force your subscription-based adware, do you still plan on supporting [`.apks`](https://developer.android.com/studio/command-line/bundletool#generate_apks)?

**Comment by archon810** [comment]
Yes, we have it on the TODO list to allow uploading .apks and .xapk files. Unfortunately, the TODO list is a mile long and is currently backed up by infrastructure issues related to bundles and otherwise.

![image](https://user-images.githubusercontent.com/469543/77592783-2eff3c80-6eb0-11ea-92ba-df52a1106d43.png)
![image](https://user-images.githubusercontent.com/469543/77592801-37577780-6eb0-11ea-96e6-176089dac911.png)
![image](https://user-images.githubusercontent.com/469543/77592843-548c4600-6eb0-11ea-96e6-204607fe2cbe.png)
